src/treetensornetworks/projttns/projttn.jl

```diff
@@ -55,3 +39,42 @@ function make_environment!(P::ProjTTN{V}, psi::TTN{V}, e::NamedEdge{V})::ITensor
   )
   return env
 end
+
+function make_environment(P::ProjTTN{V}, psi::TTN{V}, e::NamedEdge{V})::ITensor where {V}
```

Comment:
```suggestion
function make_environment(P::ProjTTN, psi::TTN, e::NamedEdge)
```

Reply:
I removed the type restriction in the newly introduced functions, but didn't touch what already existed in that style.